extensions/cursor-agents/src/tools/get-repositories.ts

```diff
@@ -0,0 +1,22 @@
+import { Cache } from "@raycast/api";
+
+type Repository = {
+  id: string;
+  name: string;
+  url: string;
+};
+
+/**
+ * Returns all available repositories that the user has configured in Cursor Agents.
+ * This can be used to determine which repository the user might be referring to.
+ */
+export default async function tool() {
+  const cache = new Cache();
+  const repositoriesData = cache.get("repositories");
+
+  if (!repositoriesData) {
+    return "No repositories have been configured yet. User needs to add repositories first through the Cursor Agents extension.";
```

Comment:
repo can only be added via the "Launch Agent" command right? might be good to clarify that

or, a new tool to add a repo based on a URL, so the tool can reply back asking for user to provide a URL and add it, would this be possible?

Reply:
Oh yeah good idea with the new tool. Let me add athat